frontend/src/templates/NavigationPrompt/IsDirtyNavigationPrompt.tsx

```diff
@@ -0,0 +1,12 @@
+import {
+  isDirtySelector,
+  useDirtyFieldStore,
+} from '~features/admin-form/create/builder-and-design/useDirtyFieldStore'
+
+import { NavigationPrompt } from './NavigationPrompt'
+
+export const IsDirtyNavigationPrompt = () => {
```

Comment:
nit: rename to `DirtyNavigationPrompt`? should we also export it from `index.ts` for cleaner imports?

Reply:
ok